src/injected/visualization/svg-drawer.ts

```diff
@@ -228,6 +236,7 @@ export class SVGDrawer extends BaseDrawer {
     }
 
     private createLinesInTabOrderVisualization(
+        items: TabbedItem[],
```

Comment:
Are we adding this parameter to be clearer about which TabbedItem array we are using?

Reply:
Yeah, technically this doesn't need to happen in this PR but I have a refactor that will make use of the fact that it is agnostic to which array is used.